config/eslint/eslintrc.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
     "@typescript-eslint",
   ],
   rules: {
+    "@nomiclabs/only-hardhat-plugin-error": "off",
```

Comment:
I'm a bit confused about this line. I'd say it's not necessary. Maybe you added it while debugging something?

Reply:
My assumption was that all rules were "on" by default when using a plugin that exports multiple rules, so this would trigger in the hardhat repo unless I turned it off. That may be an incorrect assumption, I'm not sure.

Reply:
I don't think so, as we explicitly enable it here: https://github.com/NomicFoundation/hardhat/blob/main/packages/hardhat-core/.eslintrc.js#L8

Reply:
OH I missed that entirely. Okay I'll remove that line.